Lib/asyncio/__main__.py

```diff
@@ -75,12 +77,36 @@ def run(self):
                 f'Use "await" directly instead of "asyncio.run()".\n'
                 f'Type "help", "copyright", "credits" or "license" '
                 f'for more information.\n'
-                f'{getattr(sys, "ps1", ">>> ")}import asyncio'
```

Comment:
Why delete this line? It's there to remind occasional/first-time users that there's no need to start each session with `import asyncio`.

Reply:
Good point. We need to recreate it in a smarter way to include optional colorization.

Reply:
Done.